examples/soap-calculator/src/controllers/calculator.controller.ts

```diff
@@ -0,0 +1,69 @@
+import {inject} from '@loopback/core';
+import {get, param, HttpErrors} from '@loopback/rest';
+
+import {
+  CalculatorService,
+  CalculatorParameters,
+  AddResult,
+  MultiplyResult,
+  DivideResult,
+  SubtractResult,
+} from '../services/calculator.service';
+
+export class CalculatorController {
+  constructor(
+    @inject('services.CalculatorService')
+    protected calculatorService: CalculatorService,
+  ) {}
+
+  @get('/multiply/{arg1}/{arg2}')
+  async multiply(
+    /*
+    * we are using integer here so that the coercion system in our rest server
```

Comment:
Should there be 1 test to test a non-integer value? I saw tests for missing value and correct value.

Reply:
Ok, I agree to extend the test to this scenario. As for the image, I was thinking to link to the one I will include in the tutorial. Do you want me to place it in the docs/img folder ?

Reply:
I added in the acceptance test.